src/pages/Login.js

```diff
@@ -133,11 +133,11 @@ class LoginPage extends Component {
             <div className='content-right col-sm-4 col-sm-offset-1'>
               <div className='header'>{translate('login.header')}</div>
               <form onSubmit={this.handleLogin}>
-                <input type='text' className='rounded-corner' required='true' autoFocus='true'
+                <input type='text' className='rounded-corner' required={true} autoFocus={true}
```

Comment:
this reads kinda weird, required and autoFocus are the reactJs interpretations of the boolean `true` ?

Reply:
I fixed it because sometimes I got error/warning complains about that string...it expects boolean. updating the boolean stops the complains.